src/main/java/com/dougnoel/sentinel/system/DownloadManager.java

```diff
@@ -178,6 +198,16 @@ public static boolean verifyPDFContent(URL url, String expectedText, int pageSta
         }
         pdfStripper.setStartPage(pageStart);
         pdfStripper.setEndPage(pageEnd);
+
+        try {
+            pdDoc = PDDocument.load(file);
+        } catch (InvalidPasswordException e) {
+        	var errorMessage = SentinelStringUtils.format("PDF file {} was password protected.", url.toString());
```

Comment:
Could be overkill but can we pass the password to open it, if we know it?

Reply:
I actually don't know. I can check on that later

Reply:
you actually can but I'm not going to implement it now